oeps/processes/oep-0010-proc-openedx-releases.rst

```diff
@@ -37,7 +37,7 @@ process.
 Motivation
 **********
 
-Open edX releases happen roughly every six months. EdX engineers are typically
+Open edX releases happen roughly every six months. Open edX engineers are typically
```

Comment:
@feanil This change doesn't jive with the following sentence.

Reply:
Good spotting thanks.  I think I'll drop the mention of edx.org entirely.